homeassistant/components/tuya/light.py

```diff
@@ -18,6 +18,9 @@
 from .const import DOMAIN, TUYA_DATA, TUYA_DISCOVERY_NEW
 
 PARALLEL_UPDATES = 0
+NORMAL_LIGHT_RANGE = (1, 255)
+TUYA_LIGHT_TURN_ON_RANGE = (28, 255)
+TUYA_LIGHT_RANGE = (10, 1000)
```

Comment:
This accept decimal, probably right range is `(10.3, 1000)`

Reply:
```suggestion
TUYA_LIGHT_RANGE = (10.3, 1000)
```